### ISSUE TITLE
403 ForbiddenError when creating JobCollection

### ISSUE THREAD

**Issue by hamish-tushar** [Azure/azure-sdk-for-net#3463]
The api call for job collection creation is failing.

var jobCollectionCreateResponse = schedulerMgmtClient.JobCollections.Create(cloudServiceName, jobCollectionLabel, jobCollectionCreateParameters); is failing with 

<Error xmlns=\"http://schemas.microsoft.com/windowsazure\" xmlns:i=\"http://www.w3.org/2001/XMLSchema-instance\">
<Code>ForbiddenError</Code>
<Message>The resource is blocked to recieve any updates.</Message>
</Error>

Adding  jobs to existing collection works just fine, but creating a new collection is failing. Any help to debug, log more will be appreciated. Help appreciated

**Comment by shahabhijeet** [comment]
@pinwang81 could you take a look at this please.

**Comment by Sumanan** [comment]
Hi Hamish-tushar,
Could you please provide us with your subscription id?

Thanks,
Sumanan Jeya

**Comment by hamish-tushar** [comment]
ddb35fb8-331c-4de2-b509-197eeda1996b

**Comment by hamish-tushar** [comment]
Any updates on this?

**Comment by hamish-tushar** [comment]
hei guys, can we know whats going on. WE have a prod system thats waiting on this.

**Comment by AlexGhiondea** [comment]
Given that it has been a long time I am going to close this issue for now.

If you are still running into issues please file a new issue and we will take a look.
